src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -474,10 +497,24 @@ Environment variable: `$WEB_EXT_TIMEOUT`
 
 #### `--id`
 
-A custom identifier string for the extension. This has no effect if the extension already declares an identifier in its manifest. This option may be useful for signing versions of an exisiting extension that you own.
+A custom identifier string for the extension. This has no effect if the extension already declares an identifier in its manifest. This option may be useful for signing versions of an existing extension that you own. 
+
+::: note alert
+This variable cannot be used with the add-on submission API - custom identifers must be specified in the manifest.
+:::
 
 Environment variable: `$WEB_EXT_ID`
 
+#### `--amo-metadata`
+
+Path to a JSON file containing an object with metadata to be passed to the add-on submission API. Typically this is used to submit the required metadata for the first listed version of an extension (e.g. `categories`; `license`), but any [supported JSON metadata](https://addons-server.readthedocs.io/en/latest/topics/api/addons.html) can be supplied. 
+
+::: note alert
+This variable is not used with the signing API.
```

Comment:
```suggestion
This option is only used when combined with `--use-submission-api`.
```

Reply:
I'm going to merge the suggestion, but I note that `--amo-base-url` doesn't use the same terminology about combining options.